app/services/pipeline_report_service.rb

```diff
@@ -259,23 +257,10 @@ def get_pipeline_status(pipeline_run)
       errorMessage: pipeline_run.error_message,
       knownUserError: pipeline_run.known_user_error,
       jobStatus: pipeline_run.job_status_display,
-      pipelineRunReportAvailable: pipeline_run_report_available?(pipeline_run),
+      reportAvailable: pipeline_run && pipeline_run.report_ready?,
```

Comment:
sidenote: I prefer us to use consistent names in JS and ruby for traceability unless there is a good reason not to. why not `reportReady`?